packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -18,10 +22,39 @@ MeetingUtil.parseLocus = (response) => {
   return parsed;
 };
 
+MeetingUtil.remoteUpdateAudioVideo = (audio, video, meeting) => {
+  if (meeting) {
+    const localMedias = Media.generateLocalMedias(meeting.mediaId, audio, video);
+    if (isEmpty(localMedias)) {
+      return Promise.reject(new Error('You need a media id on the meeting to change remote audio.'));
+    }
+    return meeting.meetingRequest.remoteAudioVideoToggle({
+      locusUrl: meeting.locusUrl,
+      selfId: meeting.selfId,
+      localMedias,
+      deviceUrl: meeting.deviceUrl,
+      correlationId: meeting.correlationId
+    });
+  }
+  return Promise.reject(new Error('You need a meeting object.'));
```

Comment:
Please move this to the top of the method and return early. That removes a level of indentation and you don't have to go back up in the method to see what case hits here.

Reply:
I did, i wonder why it didn't get included